src/arkouda_server.chpl

```diff
@@ -265,13 +265,24 @@ proc main() {
             if (trace) {
               try {
                 if (cmd != "array") {
+                  var size: int = args.size;
+                  var n: int = 12;
+                  // if args.size > 2*n, only first n characters and last n characters are logged
+                  if (size > 2*n) {
+                    asLogger.info(getModuleName(), getRoutineName(), getLineNumber(),
+                                  ">>> %t %t...%t".format(cmd, args[0..#n], args[size-n..#n]));
+                  }
```

Comment:
Note: I can change this to the following 
```python
if (size > 2*n) {
  asLogger.info(getModuleName(), getRoutineName(), getLineNumber(),
                         ">>> %t %t".format(cmd, args[0..#n]+'...'+args[size-n..#n]));
}
```
which results in the logging output being only 2 arguments instead of 3

```python
pd = ak.array([1,2,3])
pd.register('reallyreallyreallyreallyreallyreallyreallyreallyreallyreallyreallyreallyreallylongname')

2021-07-28:14:55:41 [arkouda_server] main Line 272 INFO [Chapel] >>> "register" "id_1 reallyr...allylongname"
...
2021-07-28:14:55:41 [arkouda_server] main Line 272 INFO [Chapel] >>> "repr" "reallyreally...longname 100"
```

Reply:
I changed this section to have this functionality and minimize diff with master by using a ternary declaration of `short_args` and eliminating the need for an extra if-else